Kinvey/Kinvey/Persistable.swift

```diff
@@ -93,6 +94,40 @@ public func <- <Transform: TransformType>(left: inout Transform.Object!, right:
     kinveyMappingType(left: right.0, right: right.1.currentKey!)
     left <- (right.1, right.2)
 }
+  
+class ListValueTransform<T: RealmSwift.Object>: TransformOf<List<T>, [JsonDictionary]> where T: BaseMappable {
```

Comment:
That's fine. For public docs, it'll be nice to prescribe one way of writing transforms.